Chakra UI CLI  v1.2.0 Generate theme typings problem with variants

I'm trying to generate typings with the latest version of chakra 1.3 and Chakra UI CLI  v1.2.0.
but I'm getting an error 

`error TS2322: Type '{ baseStyle: { fontFamily: string; fontSize: string; fontWeight: number; textTransform: string; }; sizes: { sm: { fontSize: string; }; md: { fontSize: string; }; lg: { fontSize: string; }; }; variants: { outline: (props: any) => { ...; }; ... 5 more ...; linkSecondary: (props: any) => { ...; }; }; defaultProps: { .....' is not assignable to type 'DeepThemeExtension<Partial<{ baseStyle: { lineHeight: string; borderRadius: string; fontWeight: string; _focus: { boxShadow: string; }; _disabled: { opacity: number; cursor: string; boxShadow: string; }; _hover: { _disabled: { ...; }; }; }; variants: { ...; }; sizes: { ...; }; defaultProps: { ...; }; }>, ComponentSt...'.`

## 💥 Steps to reproduce

```js
export const Button = {
  variants: {
    outline: (props: any) => {
      const { colorScheme: c } = props;
      return {
        borderColor: `${c}.500`,
        color: `${c}.500`,
        _hover: {
          bg: `${c}.500`,
          borderColor: `${c}.500`,
          color: "white"
        }
      };
    },
  }
};
```

chakra-cli tokens src/theme/index.ts

## 🧐 Expected behavior

if you do this

```js
export const Button = {
  variants: {
    outline: {
        borderColor: `red.500`,
        color: `red.500`,
        _hover: {
          bg: `red.500`,
          borderColor: `red.500`,
          color: "white"
        }
     }
  }
};
```

works.
it also works in chakra 1.2 with Chakra UI CLI  v1.1.0

--- a/.changeset/red-mayflies-reflect.md
+++ b/.changeset/red-mayflies-reflect.md
@@ -1,0 +1,5 @@
+---
+"@chakra-ui/cli": patch
+---
+
+CLI tokens command now ignores TS errors in your theme.ts file

--- a/tooling/cli/bin/tsconfig.json
+++ b/tooling/cli/bin/tsconfig.json
@@ -5,5 +5,8 @@
     "allowJs": true,
     "strict": false,
     "noImplicitAny": false
+  },
+  "ts-node": {
+    "transpileOnly": true
   }
 }

